packages/studio-ui/src/web/components/Content/Select/index.tsx

```diff
@@ -296,14 +311,17 @@ class SelectContent extends Component<Props, State> {
           onChange={this.onSearchChange}
           autoFocus
           value={this.state.searchTerm}
+          onKeyDown={this.onKeyDown}
         />
         <hr />
         <div className="list-group">
           {categories.map((category, i) => (
             <a
               key={i}
               onClick={() => this.setState({ newItemCategory: category, newItemData: null })}
-              className={`list-group-item list-group-item-action ${style.createItem}`}
+              className={`list-group-item list-group-item-action ${style.createItem} ${
+                thereIsASearchResult ? '' : i === this.state.activeItemIndex ? 'active' : ''
```

Comment:
Please avoid using ? : twice on the same line. This makes it harder to understand the logic. Instead, you can move them to a named variable and use those

Also, I would suggest a diferent naming: hasSearchResults